frontend/src/components/generateQR/GenerateQR.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import {Box, Card, CardContent, Typography} from "@mui/material";
+import QrCodeIcon from '@mui/icons-material/QrCode';
+import {Titles} from "../../common/constansts/titles";
+import "../../common/styles/icons.css";
+import "./styles.css";
+import "../../common/styles/container.css";
+import {GenerateQRProps} from "./props";
+
+const GenerateQr: React.FC<GenerateQRProps> = ({qrCode}) => {
```

Comment:
Name components correctly, they are nouns
```suggestion
const QrGeneration: React.FC<GenerateQRProps> = ({qrCode}) => {
```